Add manufacturer to shopware api

TimBlommestijn-flooris opened the pull request:
### changes
- added manufacturer client & model
- added import for PaymentMethodClient in ShowareApi

TMBL-DEV committed:
Add product-manufacturer endpoint

- created manufactureModel & client
- added product-manufacturer to searchClient

TMBL-DEV committed:
Added needed use statements in ShopwareApi

- imported PaymentMethodClient
- imported ManufactureClient

fkwakkenbos commented on src/ShopwareApi.php:
```diff
@@ -17,10 +17,12 @@
 use Flooris\ShopwareApiIntegration\Clients\ProductMediaClient;
 use Flooris\ShopwareApiIntegration\Clients\PropertyGroupClient;
 use Flooris\ShopwareApiIntegration\Clients\CalculatedTaxClient;
+use Flooris\ShopwareApiIntegration\Clients\PaymentMethodClient;
```
Wordt deze gebruikt? Ik zie voor de rest geen verwijzing naar de PaymentMethodClass in deze pull-request.

fkwakkenbos commented on src/Clients/ManufactureClient.php:
```diff
@@ -0,0 +1,86 @@
+
+    public function list($limit = 25, $page = 1): stdClass
+    {
+        return $this->getShopwareApi()->search()->products(limit: $limit, page: $page);
```
Moet dit niet `manufacturers` zijn i.p.v. `products` ?

TMBL-DEV committed:
ManufacturerClient fixed list function

- changed shopware search to manufacturers instead of products in the list function from manufacturerClient

fkwakkenbos commented on src/Clients/ManufactureClient.php:
```diff
@@ -0,0 +1,86 @@
+
+    public function baseUri(): string
+    {
+        return 'product-manufacturer';
```
Strange URI for the Manufacturer model...
